docs/modules/general/pages/architecture-overview/remoteComponents.adoc

```diff
@@ -1,41 +1,60 @@
 
 == OneCX Slot
-Every Remote Component requires a host to be displayed within a page. OneCX Slots are the elements fulfilling this role. To use Remote Component is necessary to first implement and expose it via a Microfrontend remote entry file and then create and confiure OneCX Slot in which it should be displayed.
+Every Remote Component requires a host to be displayed within a page. OneCX Slots are the elements fulfilling this role. To use a Remote Component it is necessary to first implement and expose it via a Microfrontend remote entry file and then create and confiure a OneCX Slot in which it should be displayed.
 
 === OneCX Slot definition
 OneCX Slot is a standard component used in the html content templates. It uses current workspace information to determine what content should be loaded inside the Slot. Slots themselves do not have any content and their only purpose is to render Remote Components that are assigned to them. With OneCX Slots it is easy to reuse components and dynamically change the content displayed on the pages.
 
 === Slot mechanism
-Whenver it is desired to display Remote Component, a Slot is needed to act as a host element for it. For OneCX Core Applications, there are existing Slots, which can be used to configure displayed content (especially Shell Application's slots).
+Whenever it is desired to display Remote Component, a Slot is needed to act as a host element for it. For OneCX Core Applications, there are existing Slots, which can be used to configure displayed content (especially Shell Application's Slots).
 
 To create a Slot the following requirements have to be fullfield:
-* Register Slot in Application Store: Slot needs to be registered for an Application that will be using it via the Application Store Application.
-* Use Slot in html template: Slot needs to be used in the html template of Application Microfrontend via Module page or Remote Component
-** Unique name: Slot requires to have a unique name property to be recognizable
+
+* Register Slot in Application Store: Slot needs to be registered via the Application Store.
+* Use Slot in html template: Slot needs to be used in the html template of Microfrontend's content
+* Unique name: Slot requires to have a unique name property to be recognizable
 
 To assign a Remote Component to a Slot the following requirements have to be fullfield:
+
 * Prerequisites
-** Slot has to be created and registered for the consumer Application
-** Remote Component has to be created and exposed by the provider Application
+** Slot has to be created and registered
```

Comment:
Slot has to be created ~~and registered~~ (registration in Application Store is part of the Slot creation already mentioned in previous section)